### ISSUE TITLE
Bug: `ThrowInt32OverflowException` happens in dashboard

### ISSUE THREAD

**Issue by dran1x** [0x5bfa/FluentHub#470]
### Before you start

- [X] Have you updated FluentHub? You might be using an old version.
- [X] Have you checked if a similar issue has already been reported?

### What version of Windows is this issue present?

Windows 11

### What version/architecture of FluentHub are you on?

0.8.0.60 X64 RELEASE

### Description

I installed the application through the Microsoft Store, and once I ran it and signed in with my GitHub account, it started pulling my repositories, then an error showed up on the screen.

### Steps To Reproduce

1. Install application.
2. Open and sign in.
3. Once it starts pulling your repositories it should error.

### Expected behavior

Pull all repositories and then show dashboard.

### Stack Trace

<details><summary>Stack trace details</summary><div>

```
   at System.Convert.ThrowInt32OverflowException()
   at System.Convert.ToInt32(Int64 value)
   at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1437
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1492
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1492
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1492
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1519
   at Octokit.SimpleJson.DeserializeObject[T](String json, IJsonSerializerStrategy jsonSerializerStrategy) in /_/Octokit/SimpleJson.cs:line 596
   at Octokit.Internal.JsonHttpPipeline.DeserializeResponse[T](IResponse response) in /_/Octokit/Http/JsonHttpPipeline.cs:line 62
   at Octokit.Connection.Run[T](IRequest request, CancellationToken cancellationToken, Func`2 preprocessResponseBody) in /_/Octokit/Http/Connection.cs:line 696
   at Octokit.ApiConnection.GetPage[TU](Uri uri, IDictionary`2 parameters, String accepts, ApiOptions options, Func`2 preprocessResponseBody) in /_/Octokit/Http/ApiConnection.cs:line 672
   at Octokit.ApiConnection.<>c__DisplayClass19_0`1.<<GetAll>b__0>d.MoveNext() in /_/Octokit/Http/ApiConnection.cs:line 230
--- End of stack trace from previous location ---
   at Octokit.ApiPagination.GetAllPages[T](Func`1 getFirstPage, Uri uri) in /_/Octokit/Clients/ApiPagination.cs:line 23
   at FluentHub.Octokit.Queries.Users.ActivityQueries.GetAllAsync(String login)
   at FluentHub.App.ViewModels.Viewers.DashBoardViewModel.LoadHomeContentsAsync()
   at FluentHub.App.ViewModels.Viewers.DashBoardViewModel.LoadUserHomePageAsync()
```

</div></details>

### Assets

![image](https://github.com/0x5bfa/FluentHub/assets/48390633/95ce1424-8fd5-4660-bcba-c56f85e9c391)
[Log20240406.log](https://github.com/0x5bfa/FluentHub/files/14895894/Log20240406.log)

**Comment by just-a-hriday** [comment]
Happening to me as well. Seems like a nice app but I can never get past this screen.

**Comment by 0x5bfa** [comment]
Thank you for the feedback!
This looks like sever issue.

Could you check:
- You can see other pages such as your profile page
- Exception is the same if you reload or restart the app

**Comment by beecho01** [comment]
I to have this issue. To answer the above questions:

- You can see other pages such as your profile page
	- Profile Page loads successfully from "Your Profile" and "Overview" tab
	- Repositories page loads successfully from "Your Repositories" and "Repositories" tab
	- Stars page loads successfully from "Your Stars" and "Stars" tab
	- Issues page loads successfully from "Issues" tab
	- Pull requests page loads from "Pull requests" tab
	- Discussions page loads from "Discussions" tab
	- Projects page loads from "Projects" tab
	- Organizations page loads successfully from "Your Organizations" and Organization" tab
	- Followers page loads from "Followers" tab
	- Following page loads from "Following" tab

- Exception is the same if you reload or restart the app
	- Restart, the issue appears again.
	- Uninstall and re-install, the issue appears again.

This issue appears to be solely related to the "Dashboard" page on application load and appears when repositories have appeared to fill on the sidebar.

```2024-06-12 11:05:52.743 +01:00 [ERR] LoadHomeContentsAsync
System.OverflowException: Value was either too large or too small for an Int32.
   at System.Convert.ThrowInt32OverflowException()
   at System.Convert.ToInt32(Int64 value)
   at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1437
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1492
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1492
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1492
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1519
   at Octokit.SimpleJson.DeserializeObject[T](String json, IJsonSerializerStrategy jsonSerializerStrategy) in /_/Octokit/SimpleJson.cs:line 596
   at Octokit.Internal.JsonHttpPipeline.DeserializeResponse[T](IResponse response) in /_/Octokit/Http/JsonHttpPipeline.cs:line 62
   at Octokit.Connection.Run[T](IRequest request, CancellationToken cancellationToken, Func`2 preprocessResponseBody) in /_/Octokit/Http/Connection.cs:line 696
   at Octokit.ApiConnection.GetPage[TU](Uri uri, IDictionary`2 parameters, String accepts, ApiOptions options, Func`2 preprocessResponseBody) in /_/Octokit/Http/ApiConnection.cs:line 672
   at Octokit.ApiConnection.<>c__DisplayClass19_0`1.<<GetAll>b__0>d.MoveNext() in /_/Octokit/Http/ApiConnection.cs:line 230
--- End of stack trace from previous location ---
   at Octokit.ApiPagination.GetAllPages[T](Func`1 getFirstPage, Uri uri) in /_/Octokit/Clients/ApiPagination.cs:line 23
   at FluentHub.Octokit.Queries.Users.ActivityQueries.GetAllAsync(String login)
   at FluentHub.App.ViewModels.Viewers.DashBoardViewModel.LoadHomeContentsAsync()
   at FluentHub.App.ViewModels.Viewers.DashBoardViewModel.LoadUserHomePageAsync()```

**Comment by 0x5bfa** [comment]
Thank you for the detailed information.
I confirmed this has been fixed in the latest API package.

I'll confirm if really fixed by seeing those pages you listed 🙏

**Comment by beecho01** [comment]
Thank you for the quick reply. Look forward to the new release.

**Comment by 0x5bfa** [comment]
This has been fixed in the latest build but not published to MS Store due to lack of a pipeline to do so. But we'd like to do asap.

**Comment by Lamparter** [comment]
Hello! This issue has been fixed in v0.9 which is now available in the Store 🙂
